src/DeviceStatusPage/DeviceStatusEventsCard.tsx

```diff
@@ -34,6 +34,7 @@ const DeviceStatusEventsCard = ({ device }: DeviceStatusEventsCardProps): React.
     { label: 'First registration', value: device.firstRegistration?.toLocaleString() },
     { label: 'Last connection', value: device.lastConnection?.toLocaleString() },
     { label: 'Last disconnection', value: device.lastDisconnection?.toLocaleString() },
+    { label: 'Deletion in progress', value: device.deletionInProgress },
```

Comment:
If I enter the page of a regular device, I will see a `Deletion in progress` label without further details.
That would make me question whether the device is being deleted  or not.

I would remove this label.
The user can still see whether the device is being deleted by looking at the Status field.